workflows_core/dataset/dataset.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, api: API, dataset_id: str):
 
     def __getitem__(self, index: str) -> Field:
         if isinstance(index, str):
-            if index.startswith("_cluster_"):
+            if index.startswith(("_cluster_", "_cluster_otm_")) or "_cluster_id_" in index:
```

Comment:
can this just be `if index.startswith("_cluster"):`

Reply:
I think its better to be explicit until we don't have to be